ci-operator/config/openshift/dynamic-plugin-sdk/openshift-dynamic-plugin-sdk-main.yaml

```diff
@@ -29,6 +29,20 @@ tests:
           cpu: 100m
           memory: 4Gi
       timeout: 30m0s
+    - as: e2e
+      commands: ./test-prow-e2e.sh
+      credentials:
+      - mount_path: /tmp/secret
```

Comment:
This will break the test as it tries to mount to the same folder as previous step. You could rename it to `/tmp/secret-e2e`

Reply:
@karelhala We don't need any secrets for E2E. I think I'll remove this for now.